src/backend/routers/iep.ts

```diff
@@ -271,6 +271,18 @@ export const iep = router({
                 "trial_data.notes",
                 "trial_data.created_at",
               ])
+              .select((eb) =>
```

Comment:
🐧 Above, there's one `select` that includes columns and the `jsonArrayFrom`. Here, it's separated into two `select`s. Looks like either way works, but for the sake of consistency/readability, I think we should stick with one way of writing these.

Reply:
62f4a46